lib/tasks/active_stash.rake

```diff
@@ -23,7 +24,7 @@ end
 namespace :active_stash do
   desc "Login to stash workspace"
   task :login, [:workspace] do |task, args|
-    CipherStash::Client::Profile.create(ENV.fetch("CS_PROFILE_NAME", "default"), ActiveStash::Logger, workspace: args[:workspace])
+    CipherStash::Client::Profile.create(ENV.fetch("CS_PROFILE_NAME", "default"), Rails.logger, workspace: args[:workspace])
```

Comment:
Why did this change? `ActiveStash:Logger` wraps the Rails logger but allows you to override it. Though it should actually be `ActiveStash::Logger.instance`.

Reply:
The methods on `ActiveStash::Logger` don't implement the complete `Logger` interface, which StashRB takes advantage of (specifically, passing progname as the argument and rendering the message in a block).  I'll switch to using `ActiveStash::Logger.instance`, although IMO it should be `s/STDOUT/$stderr/`, and probably have a class-level `attr_accessor :logger` (or perhaps `def instance=(l); @logger = l; end`?), for full flexibility to override.